code/__DEFINES/crafting.dm

```diff
@@ -15,6 +15,7 @@
 #define CAT_BREAD	"Breads"
 #define CAT_BURGER	"Burgers"
 #define CAT_CAKE	"Cakes"
+#define CAT_DRINK	"Drinks"
```

Comment:
you add a drinks category, but the vox wine recipe is in `CAT_MISC`?

Reply:
wanted to add that but didn't work out, guh